### PR TITLE
Adds explicit driver version

### PR DESCRIPTION
**Pull request by nvollmar**

### CODE REVIEW

File: .github/workflows/format.yml
Comment (mdedetrich) [inline .github/workflows/format.yml:12]: Although this is still a draft PR, why is this being changed? I prefer sticking to `ubuntu-latest` because at least that way we can pick up any potential issues and isolate them

Comment (nvollmar) [reply .github/workflows/format.yml:12]: I tried to keep it in sync across the project. I usually prefer fixed versions to have builds consistent. It is much harder to track sudden build failure if the change is not obvious and deliberate.

**Comment by mdedetrich** [comment]
@nvollmar @pjfanning Is this something that is slated for 1.0.0?

**Comment by nvollmar** [comment]
@mdedetrich I'll update the PR, would find it beneficial to update the driver

**Comment by nvollmar** [comment]
Also created a PR in pekko-connectors to keep them in sync: https://github.com/apache/incubator-pekko-connectors/pull/100

We've overridden the driver version of persistence-cassandra for a long time to get some critical bug fixes. Akka also upgraded to 1.14.1 meanwhile. The biggest change in 1.15.0 was CodecNotFoundException now extending DriverException, otherwise just small improvements and doc changes.

**Review by mdedetrich — APPROVED** [review]
So I am assuming that if the bugs in the newest Cassandra where critical enough that you were forced to update the version it makes sense to approve/merge this.

I will approve the PR, but I will wait for a week for a response from @pjfanning before merging.

**Comment by nvollmar** [comment]
With the upgrade we fixed a bug in akka-persistence-cassandra, which we meanwhile have also fixed directly. But the driver has other bugfixes and improvements that are beneficial.

**Review by pjfanning — APPROVED** [review]
lgtm

**Pull request merged**
